libkbfs/mdcache.go

```diff
@@ -111,3 +117,30 @@ func (md *MDCacheStandard) MarkPutToServer(
 	rmd.putToServer = true
 	md.lru.Add(key, rmd)
 }
+
+// GetIDForHandle implements the MDCache interface for
+// MDCacheStandard.
+func (md *MDCacheStandard) GetIDForHandle(handle *TlfHandle) (tlf.ID, error) {
+	md.lock.RLock()
+	defer md.lock.RUnlock()
+	key := handle.GetCanonicalPath()
+	tmp, ok := md.idLRU.Get(key)
+	if !ok {
+		return tlf.NullID, errors.WithStack(NoSuchTlfIDError{handle})
```

Comment:
Is a stack here useful for debugging? I wonder how costly `runtime.Callers` is, since we'd get a cache miss here at beginning, for every TLF we try to get the ID for.

Reply:
Ok, I'll remove it.